src/functions-reference/array_operations.Rmd

```diff
@@ -334,7 +335,7 @@ Any mismatches will cause an error to be thrown.
  x3 = append_array(x1, x2);
 ```
 
-## Sorting functions {#sorting-functions}
+## Sorting Functions {#sorting-functions}
```

Comment:
This should be `Sorting functions` not `Sorting Functions`.  

* `# XXX` uses title punctuation for `XXX` (all caps other than "function" words [prepositions, determiners, auxliary verbs, other particles, etc.])
* `## XXX` should use sentence punctuation for `XXX` (initial caps, not all caps)